core/src/main/java/com/netflix/conductor/validations/WorkflowTaskTypeConstraint.java

```diff
@@ -98,11 +98,43 @@ public boolean isValid(WorkflowTask workflowTask, ConstraintValidatorContext con
                 case TaskType.TASK_TYPE_JSON_JQ_TRANSFORM:
                     valid = isJSONJQTransformTaskValid(workflowTask, context);
                     break;
+                case TaskType.TASK_TYPE_SIMPLE:
+                    valid = isSimpleTaskValid(workflowTask, context);
+                    break;
             }
 
             return valid;
         }
 
+        private boolean isSimpleTaskValid(WorkflowTask workflowTask, ConstraintValidatorContext context) {
```

Comment:
My mistake. I was thinking of adding this capability to only `SIMPLE` tasks but I guess it could be added to other task types, for example the Http task. What would you prefer? Should I add the validations to other tasks? If so, which tasks do you think should be added and which tasks should not be supported?